byterun/weak.c

```diff
@@ -31,52 +31,84 @@ value caml_ephe_list_head = 0;
 static value ephe_dummy = 0;
 value caml_ephe_none = (value) &ephe_dummy;
 
+#define CAMLassert_valid_ephemeron(eph) do{                                 \
+    CAMLassert (Is_in_heap (eph));                                          \
+    CAMLassert (Tag_val(eph) == Abstract_tag);                              \
+    CAMLassert (CAML_EPHE_FIRST_KEY <= Wosize_val (eph));                   \
+}while(0)
+
+#define CAMLassert_valid_offset(eph, offset) do{                             \
```

Comment:
It seems you took care to align all the \\, but there is one more space than necessary here 😄